app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationMailer < ActionMailer::Base
+  layout 'mailer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.